mmv1/third_party/terraform/.teamcity/components/projects/reused/mm_upstream.kt

```diff
@@ -25,7 +25,7 @@ import jetbrains.buildServer.configs.kotlin.Project
 import jetbrains.buildServer.configs.kotlin.vcs.GitVcsRoot
 import replaceCharsId
 
-fun mmUpstream(parentProject: String, providerName: String, vcsRoot: GitVcsRoot, cronSweeperVcsRoot: GitVcsRoot, config: AccTestConfiguration): Project {
+fun mmUpstream(parentProject: String, providerName: String, vcsRoot: GitVcsRoot, cronSweeperVcsRoot: GitVcsRoot, config: AccTestConfiguration, cron: NightlyTriggerConfiguration): Project {
```

Comment:
Just want to confirm if I understand it correctly - Is the modification to the MM upstream config for stopping VCR sweepers on release branch test days?

Reply:
I think I just wanted to make the functions used to create nightly testing and upstream testing Projects in TeamCity consistent, in that they both require the cron trigger to explicitly passed in.

Currently the sweepers for the VCR project will run independently of what's happening in Nightly Test projects.

Am I right that the v6.0.0 testing should use the nightly test projects and not the VCR project in GCP?

Reply:
Oh, I see, it's for consistency. 

> Am I right that the v6.0.0 testing should use the nightly test projects and not the VCR project in GCP?

Yes, correct! The v6.0.0 testing should use the nightly test projects, same as nightly runs